apps/docs/content/dotnet/how-to/customize-runtime.mdx

```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
 
 import data from '@site/static/data.json';
 
-The default .NET runtime environment contains:
+
+## Customise .NET build environment
```

Comment:
Good catch with the `build` prefix, but it was supposed to be `run.os` https://docs.zerops.io/nodejs/how-to/build-pipeline#os-1 and `run.prepareCommands` https://docs.zerops.io/nodejs/how-to/build-pipeline#os-1 instead :grimacing: Keep it as a note, it's good, but the headings can stay as they were. Also keep the paragraph in build process too.